### PR TITLE
Redesign event popover Ui

### PR DESCRIPTION
**Pull request by m4dz**

### CODE REVIEW

**Comment by frankrousseau** [comment]
Thank you for the PR. Here are my first remarks:

* code lines should be < 80 chars
* adding comments on every function you changed would be nice to improve the code quality.

File: client/app/lib/popover_view.coffee [hunk 1]
```diff
@@ -28,31 +28,42 @@ module.exports = class PopoverView extends BaseView
             trigger: 'manual'
             title: @titleTemplate @getRenderData()
             html: true
-            placement: @getDirection()
+            placement: @getDirection
```
Comment (jsilvestre) [inline client/app/lib/popover_view.coffee]: why did you remove the parenthesis?

**Comment by jsilvestre** [comment]
The popover text 'to' is not localized.

File: client/app/views/templates/popover_event.jade [hunk 2]
```diff
@@ -1,31 +1,36 @@
...
+                           checked=allDay)
+        span.caption= t('All day')
 
-.line
-    input#input-desc.input(type="text", value=model.get("description"),
-        placeholder=t("summary"))
-    input#input-place.input-small(type="text", value=model.get("place"),
-        placeholder=t("Place"))
+    label
+        span.caption= t("To")
```
Comment (jsilvestre) [inline client/app/views/templates/popover_event.jade]: it's "to" in the locale files, "To" here.

**Comment by jsilvestre** [comment]
I think you could use cozy styles for input:
* no border-radius
* mouse hover: border is orange (cozy orange)
* focus: border is blue (cozy blue)

**Comment by jsilvestre** [comment]
If you open the calendar picker, then it hides the event's title. But the calendar picker cannot be hidden afterwards.

File: client/app/lib/popover_view.coffee (hunk 1 above)
Comment (m4dz) [inline client/app/lib/popover_view.coffee]: I remove parenthesis, so the `getDirection` function is called by the popover plugin at `show` invocation, not at popover instanciation. This way, we can get the `tip` element (the popover content) as first arg in the function, and use it to compute dimensions dynamically.

Bootstrap 3 introduces an `auto` placement value, but unfortunately this one isn't available in Bootstrap 2.3.1.

File: client/app/views/templates/popover_event.jade (hunk 2 above)
Comment (m4dz) [inline client/app/views/templates/popover_event.jade]: My Bad™… I primarily use the _to_ string then forget to change it afterwards :disappointed:. Thanks.

**Comment by m4dz** [comment]
@frankrousseau thanks ! I try to keep code as clear as possible. I'll keep an eye on formatting to ensure nothing exceeds.

@jsilvestre thanks for the cozy styles, I didn't see them, I'l add it. For the calendar picker, its position over the event title is a choice, but we can display it differently if you thinks it might be misunderstood. Etrangely, I haven't any problem to hide the picker (by selecting a calendar, or by clicking again on the calendar color badge). Can you give tell me what's your browser / version ?

**Comment by jsilvestre** [comment]
NVM, it's working. I don't know what happened.

File: client/app/lib/popover_view.coffee (hunk 1 above)
Comment (jsilvestre) [inline client/app/lib/popover_view.coffee]: Okay, I didn't realize bootstrap would call the function if it's a function, or use the value if it's not.

**Comment by m4dz** [comment]
Ok, last commit should fix everything (and add some little sugar :smile:)

**Comment by frankrousseau** [comment]
Here are three remarks

* Padding: there is not enough space between elements and modal border. 
* The date field should have a left margin of 10px.
* The tab thing works until you want to reach the create button. It's disapointing that the focus leaves the popup while using the tab key.

**Comment by jsilvestre** [comment]
A few more comments:
* title/location inputs are very close to each other. Maybe a margin-top on the second one could clear things up.
* french localization:
```diff
- journée entière, jusqu'à
+ journée entière, jusqu'au
```
* french localization: "à" should be uppercase "À"

**Comment by m4dz** [comment]
@frankrousseau Im'not sur eto understand why you prefer a left margin of 10px on the date field (the end date field?). The goal is to have a pseudo sentence like _To 10:30, 12/01/2015_, so a left margin might add some extra space that can be curious in the sentence, no?

**Comment by frankrousseau** [comment]
I think the problem is the comma, could your replace it by a hyphen? My problem is that I don't like to see the date field too close to the comma when it's active. 

Another problem I noticed, when the date picker is displayed could you turn the date field border in blue?

**Comment by jsilvestre** [comment]
"All day" event seems to be broken

**Comment by m4dz** [comment]
Last commits takes care of recents reviews. @jsilvestre, I definitely cannot reproduce your issue with the all-day checkbox, but I found a typo that can be the cause of it (see 3cf5603). Can you try with this last version ?

**Comment by jsilvestre** [comment]
It looks fine now.

**Comment by jsilvestre** [comment]
@frankrousseau I let you have a last look at it. It's good to merge from my point of view.

**Comment by frankrousseau** [comment]
It's ok but is stils lacks of padding between elements and modal border.

**Comment by m4dz** [comment]
Whitespace added :smile:

**Comment by frankrousseau** [comment]
Merged! Congrats for your first contribution to the Cozy project!
